app/helpers/textile_helper.rb

```diff
@@ -0,0 +1,10 @@
+module TextileHelper
+  def textile_default_data
```

Comment:
Still this method name doesn't describe what this method is for, can we please do our future selves a favour and be more thoughtful about our own naming decisions?